Public/Standalone/Set-PDFForm.ps1

```diff
@@ -1,32 +1,92 @@
 function Set-PDFForm {
-    [CmdletBinding()]
+   [CmdletBinding()]
     param(
-        [string] $SourceFilePath,
-        [string] $DestinationFilePath,
-        [System.Collections.IDictionary]$FieldNameAndValueHashTable
+        [Parameter(Mandatory)]
+        [ValidateNotNullOrEmpty()]
+        $SourceFilePath,
+        [Parameter(Mandatory)]
+        [ValidateNotNullOrEmpty()]
+        $DestinationFilePath,
+        [System.Collections.IDictionary]$FieldNameAndValueHashTable,
+        [Switch] $Flatten
     )
+    
+    $File = (Split-Path $DestinationFilePath -leaf)
```

Comment:
This and the next 2 line make me fell dirty, but I have no other ideas on how else to use a convert-path on a path that the file will not exist on..